modules/swagger-annotations/src/main/java/io/swagger/v3/oas/annotations/media/Content.java

```diff
@@ -84,4 +85,86 @@
      */
     Extension[] extensions() default {};
 
+    /**
+     * Allows to specify the dependentSchemas value providing a Schema annotations map
+     *
+     * @since 2.2.1 / OpenAPI 3.1
+     * @return the schema properties
+     */
+    @OpenAPI31
+    DependentSchema[] dependentSchemas() default {};
+
+    /**
+     * Provides the content schema related to this schema
+     *
+     * @since 2.2.1 / OpenAPI 3.1
+     * @return content schema
+     */
+    @OpenAPI31
+    Schema contentSchema() default @Schema();
+
+    /**
+     * Provides property names related to this schema
+     *
+     * @since 2.2.1 / OpenAPI 3.1
+     * @return content schema
+     */
+    @OpenAPI31
+    Schema propertyNames() default @Schema();
+
+    /**
+     * Provides the if sub schema related to this schema
+     *
+     * @since 2.2.1 / OpenAPI 3.1
+     * @return content schema
+     */
+    @OpenAPI31
+    Schema _if() default @Schema();
+
+    /**
+     * Provides the then sub schema related to this schema
+     *
+     * @since 2.2.1 / OpenAPI 3.1
+     * @return content schema
+     */
+    @OpenAPI31
+    Schema _then() default @Schema();
+
+    /**
+     * Provides the else sub schema related to this schema
+     *
+     * @since 2.2.1 / OpenAPI 3.1
+     * @return content schema
+     */
+    @OpenAPI31
+    Schema _else() default @Schema();
+
+    /**
+     * Provides a java class to be used to disallow matching properties.
+     *
+     * @return a class with disallowed properties
+     **/
```

Comment:
comment doesn't seem to be correct as it refers to "java classes"

no `@since 2.2.1 / OpenAPI 3.1`